packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -30,7 +30,7 @@ export abstract class InstrumentationBase<T = any>
   implements types.Instrumentation {
   private _modules: InstrumentationModuleDefinition<T>[];
   private _hooks: RequireInTheMiddle.Hooked[] = [];
-  private _enabled = false;
+  protected _enabled = false;
```

Comment:
Having a `enable` and `disable` API next to a read/write property `_enabled` looks strange. Is it really intended that derived classes just modify this flag and skip calls to `disable()`?

Reply:
The other option is to have a duplicate `isEnabled` flag for instrumentations, see https://github.com/open-telemetry/opentelemetry-js-contrib/pull/432/files#r616948268

Reply:
To my understanding a readonly property or `isEnabled()` API would be sufficient instead of having a shortcut to "partial" disable an instrumentation.

Reply:
Agree with this, changed it to `isEnabled()`